pymode/libs/logilab-common-1.4.1/logilab/common/modutils.py

```diff
@@ -579,7 +584,7 @@ def is_relative(modname, from_file):
     if from_file in sys.path:
         return False
     try:
-        find_module(modname.split('.')[0], [from_file])
+        find_spec(modname.split('.')[0], [from_file])
```

Comment:
`find_spec` returns None if it can't find it, unlike `find_module` which crashes.